src/Todoist.Net/Serialization/Converters/CommandResultConverter.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+using Todoist.Net.Models;
+
+namespace Todoist.Net.Serialization.Converters
+{
+    internal class CommandResultConverter : JsonConverter<CommandResult>
+    {
+        public override CommandResult Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            if (reader.TokenType == JsonTokenType.String)
+            {
+                if (reader.GetString() != "ok")
+                {
+                    throw new JsonException("Invalid format for CommandResult object.");
```

Comment:
Let's put the value in the ctor

Reply:
Do you mean the ctor of `CommandResultConverter` or the `CommandResult` model?

Reply:
The result of `reader.GetString()`, in this case we will know what value we unable to handle.

Reply:
Ok, here's our new exception:
```c#
var stringValue = reader.GetString();
if (!string.Equals(stringValue, CommandResult.SuccessValue, StringComparison.OrdinalIgnoreCase))
{
    throw new JsonException($"Unable to parse the value '{stringValue}' to a {nameof(CommandResult)} object.");
}
```